Makefile.core.mk

```diff
@@ -76,21 +76,12 @@ build_envoy_tsan:
 build_envoy_asan:
 	export PATH=$(PATH) CC=$(CC) CXX=$(CXX) && bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) $(BAZEL_CONFIG_ASAN) //src/envoy:envoy
 
-# Implicitly depends on build, but does not require a specific configuration
-.PHONY: wasm_include
-wasm_include:
-	cp -f $$(bazel info bazel-bin)/extensions/common/node_info_generated.h $(TOP)/extensions/common/
-	cp -f $$(bazel info bazel-bin)/extensions/common/node_info_bfbs_generated.h $(TOP)/extensions/common/
-	cp -f $$(bazel info bazel-bin)/extensions/common/nlohmann_json.hpp $(TOP)/extensions/common/
-	cp -fLR $$(bazel info bazel-bin)/external/com_github_google_flatbuffers/_virtual_includes/runtime_cc/flatbuffers $(TOP)/extensions/common/
-	cp -f $$(bazel info output_base)/external/envoy/api/wasm/cpp/contrib/proxy_expr.h $(TOP)/extensions/common/
-
-build_wasm: wasm_include
-	$(foreach file, $(shell find extensions -name build_wasm.sh), cd $(TOP)/$(shell dirname $(file)) && bash ./build_wasm.sh &&) true
-
-check_wasm:
-	export PATH=$(PATH) CC=$(CC) CXX=$(CXX) && bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) $(BAZEL_CONFIG_DEV) //src/envoy:envoy
-	./scripts/generate-wasm.sh -b
+build_wasm:
+	bazel build //extensions/stats:plugin.wasm
```

Comment:
if you want to add this to the make file, we should use `BAZEL_STARTUP_ARGS`, `CC`, `CXX` all standard variable.

Reply:
But those ENV variable, or build options do not really apply to the Wasm target, right?